osu.Game/Scoring/ScoreInfo.cs

```diff
@@ -46,6 +46,12 @@ public class ScoreInfo : RealmObject, IHasGuidPrimaryKey, IHasRealmFiles, ISoftD
         /// </remarks>
         public BeatmapInfo? BeatmapInfo { get; set; }
 
+        /// <summary>
+        /// The version of the client this score was set using.
+        /// Sourced from <see cref="OsuGameBase.Version"/> at the point of score submission.
+        /// </summary>
+        public string Version { get; set; } = string.Empty;
```

Comment:
I guess probably the only question I'd have here is - do we wanna put this in replays too (and decode it from there on import)? We can probably put this in `LegacyReplaySoloScoreInfo` with practically zero effort.

Reply:
Oh yeah. I didn't realise this was so low effort, if so then let's do it.

Reply:
Have done so in https://github.com/ppy/osu/pull/25998/commits/2baf579f7ce62429e83aa4781ec3d55320b92d09 (with test coverage in https://github.com/ppy/osu/pull/25998/commits/5ff95db02c2af9ce6e7b1d707ee88440e3df9e6e).